docs/service-options.md

```diff
@@ -14,13 +14,17 @@ module.exports.options = {
     buildLocalReference(json, baseUri, fragment, i) {
       // your naming convention
     }
+  },
+  customLevels: {
```

Comment:
Maybe it's better to use customLogLevels? In this way, it is more scoped

Reply:
I agree that customLogLevels has a more scoped name, but on the other hand I think it is better if we use the same option name of Pino library, like we are doing with other options. A developer working with Fastify and Pino would expect that.

Reply:
Ok, but at least it should be under a log field, something like:

```json
"log": {
  "customLevels": {}
}
```

Because the options are related to the server, not to the logger. Under the log, we could add also other pino related options.